docs/commands/ts.revrange.md

```diff
@@ -7,7 +7,7 @@ TS.REVRANGE key fromTimestamp toTimestamp
          [FILTER_BY_TS TS...]
          [FILTER_BY_VALUE min max]
          [COUNT count]
-         [[ALIGN value] AGGREGATION aggregator bucketDuration [EMPTY]]
+         [[ALIGN value] AGGREGATION aggregator bucketDuration [BUCKETTIMESTAMP bt] [EMPTY]]
```

Comment:
@OfirMos can we include the relevant description of `BUCKETTIMESTAMP` in the command description.

Reply:
@filipecosta90 
@LiorKogan Will fill the rest of the docs after we merge